### PR TITLE
Tool to import a place from the Production API

### PR DESCRIPTION
**Pull request by rainhead**
Based on a script by @kueda, but:

- imports recursively, recording parent
- records uuid
- uses Faraday, so we can switch to v2 API + JWT token eventually

One trick with this is that Asia cannot be imported because something is amiss with its geometry. Because the script imports a place's parents recursively, nothing in Asia can be imported. Do you have a hunch for whether this is a problem with production data or with the import script?

### CODE REVIEW

**Comment by kueda** [comment]
@rainhead yeah, unfortunately Asia has a geometry that is topologically valid in the database but doesn't pass our application-level validation test of not having impossible coordinates. The best fix would be to change the record in the database, but... that would kick off a cascade of re-indexing jobs that I don't think is warranted here.

I think this is fine to merge as-is, but if you're interested, one potential workaround is to have the script review every point in the new geometry and snap impossible values (e.g. longitude of -180.1) to possible ones (e.g. longitude of -180.0).

File: tools/import_place_from_production.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+if Rails.env.production?
```
Comment (todtb) [inline tools/import_place_from_production.rb:3]: `unless Rails.env.development?` ?

**Review by todtb — COMMENTED** [review]
Very helpful, thanks!

File: tools/import_place_from_production.rb (hunk 1 above)
Comment (kueda) [reply tools/import_place_from_production.rb:3]: Fair. No reason to be running this in a test environment.

Comment (rainhead) [reply tools/import_place_from_production.rb:3]: Sure, this was just copy-pasted from another file in the directory.

Comment (rainhead) [reply tools/import_place_from_production.rb]: Addressed in [bc29c2f](https://github.com/inaturalist/inaturalist/pull/3655/commits/bc29c2fe096d0d50047db774aa00685014d4cf20).

**Pull request merged**